c++/src/capnp/capability.h

```diff
@@ -686,6 +691,9 @@ class RequestHook {
   virtual kj::Promise<void> sendStreaming() = 0;
   // Send a streaming call.
 
+  virtual kj::Promise<void> sendRealtime() = 0;
```

Comment:
I think `sendRealtime()` isn't needed anymore with the `CallHints` approach? A `CallHints` should have been specified when creating the `RequestHook`, therefore we can just use `sendStreaming()` and it'll know whether or not to treat as realtime.